plugins/cairo-lang-macro/src/types/token.rs

```diff
@@ -95,12 +95,14 @@ impl TokenTree {
     }
 }
 
+pub type TextOffset = u32;
```

Comment:
Should we add such alias if this `TextOffset` does not match compiler one anyway?

Reply:
We don't need to structure our spans exactly the same as compiler ones - we just need to be able to convert between those two. 

I don't think structuring offsets into new types gives us any advantage - let me know if you think otherwise. 
In practice, type alias acts just as a comment, so I don't see anything bad with that.

Reply:
> I don't think structuring offsets into new types gives us any advantage - let me know if you think otherwise.

I suppose you're right, keeping it as a comment makes sense 👍🏻